go-controller/pkg/libovsdbops/model.go

```diff
@@ -227,6 +235,16 @@ func copyIndexes(model model.Model) model.Model {
 		return &sbdb.IGMPGroup{
 			UUID: t.UUID,
 		}
+	case *sbdb.Encap:
+		return &sbdb.Encap{
+			UUID:        t.UUID,
+			ChassisName: t.ChassisName,
+		}
+	case *sbdb.PortBinding:
+		return &sbdb.PortBinding{
```

Comment:
I don't know if we have a use for it, but datapath and tunnel_key are also indexes.